ui/src/app/views/account-creation/component.ts

```diff
@@ -43,6 +40,9 @@ export class AccountCreationComponent {
   gsuiteDomain: string;
   usernameCheckTimeout: NodeJS.Timer;
 
+  // TODO: Injecting the parent component is a bad separation of concerns, as
+  // well as injecting LoginComponent. Should look at refactoring these
+  // interactions.
```

Comment:
:100:

That we can do this at all is interesting, but it makes me nervous.  I don't _think_ this is an intended use case (hard to find any docs or other info on it); I think we're operating outside of Angular's support by doing this.